query-engine/driver-adapters/js/smoke-test-js/.envrc.example

```diff
@@ -1,3 +1,6 @@
+# This will load the .envrc at the root of the `prisma-engines` repository before loading this one
```

Comment:
If that was the intention, then since it's a template, we can leave it commented-out here so if anyone else except me also needs it to be inherited, they can easily so in their `.envrc`.

Reply:
Jan, I confirm that was the original intention, but it's not something worth being picky about, so it's ok to change if needed. Alexey, if you have a good enough reason to introduce this change (e.g., it'd help with setting up the CI), then please go ahead :)

Reply:
I made this commented out by default and updated the comment. WDYT?